cwms-data-api/src/test/java/cwms/cda/api/BinaryTimeSeriesControllerTestIT.java

```diff
@@ -267,11 +288,135 @@ void test_create_get_delete_get() throws IOException {
+            .queryParam(Controllers.VERSION_DATE, versionDateStr)
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .get("/timeseries/binary/")
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_OK))
+            .body("binary-values.size()", equalTo(0))
+        ;
+    }
+
+    @Test
+    void test_create_get_delete_get_lrts() throws Exception {
+
+        // Structure of test:
+        //
+        // 1)Try to create the binary time series without LRTS
+        // 2)Create the binary time series with LRTS
+        // 3)Retrieve the binary time series and assert that it exists
+        // 4)Delete the binary time series
+        // 5)Retrieve the binary time series and assert that it does not exist
+
+
+        // Step 1)
+        // Try to create the binary time series without LRTS header set
+        TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+        String tsIdentifier = "TsBinTestLoc.Flow.Inst.1HourLocal.0.lrts";
+        String tsData = getTsBodyNewLRTSInterval(tsIdentifier);
+
+        createTimeseries(OFFICE, tsIdentifier, true);
+
+        given()
+            .log().ifValidationFails(LogDetail.ALL,true)
+            .accept(Formats.JSONV2)
+            .contentType(Formats.JSONV2)
+            .body(tsData)
+            .header("Authorization", user.toHeaderValue())
+            .header(ApiServlet.IS_NEW_LRTS, false)
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .post("/timeseries/binary/")
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_INTERNAL_SERVER_ERROR));
+
+        // Step 2)
+        // Create the binary time series
+
+        given()
+            .log().ifValidationFails(LogDetail.ALL,true)
+            .accept(Formats.JSONV2)
+            .contentType(Formats.JSONV2)
+            .body(tsData)
+            .header("Authorization", user.toHeaderValue())
+            .header(ApiServlet.IS_NEW_LRTS, true)
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .post("/timeseries/binary/")
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_CREATED))
+        ;
+
+        // Step 3)
+        // Retrieve the binary time series and assert that it exists
+
+        given()
+            .log().ifValidationFails(LogDetail.ALL,true)
+            .accept(Formats.JSONV2)
+            .queryParam(Controllers.OFFICE, OFFICE)
+            .queryParam(Controllers.NAME, tsIdentifier)
+            .queryParam(Controllers.BEGIN, "2004-05-01T12:00:00Z")
+            .queryParam(Controllers.END, "2007-05-19T16:00:00Z")
+            .header(ApiServlet.IS_NEW_LRTS, true)
```

Comment:
Add a test right after this that tests without the new header to assert that the old interval id is returned